lib/utils.js

```diff
@@ -22,20 +21,18 @@ module.exports = { log }
 
 const parseSubject = (s) => {
   // TODO: find a more elegant way to capture both cases in one regex
-  let matches = s.match(T2R)
-  if (!matches || matches.some((v) => !v)) {
-    matches = s.match(T1R)
-  }
+  const matches = s.match(R)
```

Comment:
much more elegant indeed. the TODO comment can be removed then

Reply:
oki, gonna add to changelog and merge this :)